CHANGELOG.md

```
@@ -1,5 +1,10 @@
 # Change Log
 
+## [4.68.2](https://github.com/plivo/plivo-php/tree/v4.68.2) (2026-01-21)
+**Feature - HA Number support**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)